app/build.gradle

```diff
@@ -296,7 +296,10 @@ android {
     preDexLibraries = project.APTOIDE_PRE_DEX_LIBRARIES
     javaMaxHeapSize = project.APTOIDE_DEX_JAVA_MAX_HEAP_SIZE
   }
-  sourceSets { release { java.srcDirs = ['src/release/java', 'src/release/java/'] } }
+  sourceSets {
+    release { java.srcDirs = ['src/release/java', 'src/release/java/'] } 
+    vanilla {res.srcDirs = ['src/vanilla/res', 'src/vanilla/res/raw'] }
```

Comment:
this should be remove later with the local videos, we can't forget this.

Reply:
this will be fixed in a later ticket;